README.md

```diff
@@ -109,19 +109,20 @@ License
 
 This project is licensed under the terms of the [MIT license](LICENSE).
 
-[Nix]: https://nixos.org/nix/
+[Nix]: https://nixos.org/guides/how-nix-works.html
 [NixOS]: https://nixos.org/
-[Nixpkgs]: https://nixos.org/nixpkgs/
-[manual]: https://nix-community.github.io/home-manager/
+[Nixpkgs]: https://github.com/NixOS/nixpkgs
+[manual]: https://nix-community.github.io/home-manager/index.html
```

Comment:
this link isn't broken. https://nix-community.github.io/home-manager/ serves the `index.html` page

Reply:
This avoids a unnecessary refresh when you click the links in table of contents which explicitly uses `/index.html`. This is also the case for the manual links below.